data/keechain.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Keechain
+web: 
+git: https://github.com/yukibtc/keechain
+instances:
+tags:
+- signer
+platforms:
+- macOS
+- Linux
+license: MIT
+progLang:
+- Rust
+- Shell
```

Comment:
Shell=cli

```suggestion
- cli
```

Reply:
I am confused. I usually look at the bottom right "Languages" panel on GitHub to put what can be put there. 
If I put cli won't that be a platform rather than a language?

![image](https://user-images.githubusercontent.com/44260360/228112623-bfabea14-755d-4c46-b72b-a6301d8eb21c.png)